src/neo4j/neo4j-functions.js

```diff
@@ -140,4 +144,32 @@ export async function neighbourhoodWithoutComplexes(id) {
         await session.close();
     }
     return record;
+}
+
+/**
+ * Get the graph pertaining to a specific factoid document
+ * @param { String } id factoid UUID for document
+ * @returns null if document does not exist in database, array of objects with
+ * fields for the nodes and edges otherwise
+ */
+export async function get(id) { // UNTESTED
+    const driver = getDriver();
+    let session;
+    let record;
+    try {
+        session = driver.session({ database: "neo4j" });
+        let result = await session.executeRead(tx => {
+            return tx.run(giveConnectedInfoForDocument, { id: id });
+        });
+        if (result.records.length > 0) {
+            record = result.records;
+        } else {
+            record = null;
+        }
+    } catch (error) {
```

Comment:
Later versions of eslint flag these `catch` clauses as unnecessary (since they don't do anything).

Reply:
I'll take them out of that file, and make note of it for my code clean-up at the end of the term